.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,57 @@
+name: Upload Python Package
+
+on:
+  workflow_call:
+    inputs:
+      test_pypi:
+        type: boolean
+        default: false
+    secrets:
+      PYPI_PASSWORD:
+        required: true
+
+jobs:
+  upload-python-package:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          # noninteractive is necessary to install libgdal-dev
+          # which is needed for python gdal which is
+          # required by rasterio
+          export DEBIAN_FRONTEND=noninteractive
```

Comment:
rasterio is only needed for the actinia-core workflow, as this is one of the dependencies.
Either we remove all these lines and keep the actinia-core workflow independant, or we keep it here and can use this workflow also in actinia-core.